tests/dummy/config/environment.js

```diff
@@ -7,6 +7,7 @@ module.exports = function (environment) {
     rootURL: "/",
     locationType: "history",
     EmberENV: {
+      EXTEND_PROTOTYPES: false,
```

Comment:
Isn't this `false` by default for addons?

[from docu](https://api.emberjs.com/ember/release/classes/EmberENV/properties/EXTEND_PROTOTYPES?anchor=EXTEND_PROTOTYPES)
> Note, when disabled (the default configuration for Ember Addons), you will instead have to access all methods and functions from the Ember namespace.

Reply:
This is the default blueprint done by `ember-cli-update`. I highly suggest keeping it up to date with the default blueprint to make updates easier

Reply:
*Muy bien* then!